### ISSUE TITLE
Investigate build problem

### ISSUE THREAD

**Issue by louischan-oursky** [authgear/authgear-sdk-js#312]
https://github.com/authgear/authgear-sdk-js/actions/runs/9989624079/job/27608717859

**Comment by fungc-io** [comment]
@louischan Any follow up needed? build was successful?

**Comment by louischan-oursky** [comment]
Yes, we need follow up. React Native iOS build was unsuccessful https://github.com/authgear/authgear-sdk-js/actions/runs/10058356674/job/27801318045
